.bazelversion

```diff
@@ -1,7 +1 @@
-6.3.2
-# The first line of this file is used by Bazelisk and Bazel to be sure
-# the right version of Bazel is used to build and test this repo.
-# This also defines which version is used on CI.
-#
-# Note that you should also run integration_tests against other Bazel
-# versions you support.
+7.1.0
```

Comment:
Did version 7 with `--enable_bzlmod=false` cause any issues that can be fixed+merged on their own before enabling bzlmod?

Reply:
I'd prefer not to. There are errors since I'm using rules that aren't bazel 7 compatible (the version of rules_js is quite old). I'd prefer to update the deps once rather than twice if that's okay